packages/expandable/src/ux-expandable.ts

```diff
@@ -0,0 +1,84 @@
+import { customElement, useView, PLATFORM, bindable, TaskQueue, inject } from 'aurelia-framework';
+import { UxComponent, StyleEngine, normalizeBooleanAttribute } from '@aurelia-ux/core';
+import { UxExpandableTheme } from './ux-expandable-theme';
+
+@inject(Element, StyleEngine, TaskQueue)
+@customElement('ux-expandable')
+@useView(PLATFORM.moduleName('./ux-expandable.html'))
+export class UxExpandable implements UxComponent {
+  constructor(public element: HTMLElement, private styleEngine: StyleEngine, private taskQueue: TaskQueue) { }
+
+  static OPEN_CHANGED_EVENT = 'open-changed';
+
+  header: HTMLElement;
+  content: HTMLElement;
+  contentContainer: HTMLElement;
+
+  @bindable
+  public theme: UxExpandableTheme;
+  public themeChanged(newValue: any) {
+    if (newValue != null && newValue.themeKey == null) {
+      newValue.themeKey = 'expandable';
+    }
+
+    this.styleEngine.applyTheme(newValue, this.element);
+  }
+
+  public openBoolean: boolean = false;
+  @bindable
+  open: boolean | string = false;
+  openChanged() {
+    this.openBoolean = normalizeBooleanAttribute('open', this.open);
+    this.updateContainerHeight();
+    this.element.dispatchEvent(new CustomEvent(UxExpandable.OPEN_CHANGED_EVENT, { detail: { component: this, open: this.openBoolean } }));
```

Comment:
can this be in multiple lines? We don't have any rule yet but having a massive line like this is gonna be refactored anyway. Maybe?
```suggestion
    this.element.dispatchEvent(new CustomEvent(
      UxExpandable.OPEN_CHANGED_EVENT,
      { detail: { component: this, open: this.openBoolean } }
    ));
```